python/nrel/routee/compass/compass_app.py

```diff
@@ -135,6 +135,55 @@ def run(
             return results[0]
         return results
 
+    def run_in_batches(
+        self, fp: TextIO, batch_size: int
```

Comment:
Instead of passing a file pointer, let's instead pass a Path or string and then create the file pointer in this method.

Reply:
gotcha will update